app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java

```diff
@@ -152,6 +155,12 @@ public void updateFromItem(final InfoItem infoItem,
             itemPublishedTime.setText(item.getTextualUploadDate());
         }
 
+        if (item.isHeartedByUploader()) {
```

Comment:
Tip to turn this into a one liner and make it more readable:
``itemHeartView.setVisibility(item.isHeartedByUploader() ? View.VISIBLE : View.GONE);``